templates/search/blocks/search_results__list-card.html

```diff
@@ -11,92 +11,64 @@
 {% comment %} TODO downloadable, and record variables {% endcomment %}
 
 <li class="search-results__list-card" data-iaid="{{ record.iaid }}" data-score="{{ record.score }}">
+
+    <i class="search-results__icon fa-solid fa-file"></i>
+
     <div class="search-results__list-card-details">
         <h3 class="tna-heading-m">
             <a href="{% if form.group.value == bucketkeys.ARCHIVE.value %}{% record_url record level_or_archive='Archive' form_group=form.group.value %}{% else %}{% record_url record level_or_archive=record.level form_group=form.group.value %}{% endif %}" class="search-results__list-card-heading-link" data-link="{{ record.reference_prefixed_summary_title }}" data-link-type="Search results list" search-bucket="{{ buckets.current.label }}">
                 {% if record.reference_number %}
                 <span class="sr-only">Reference number {{ record.reference_number }}: </span>
                 {% endif %}
                 {{ record.summary_title }}
+                {# API data required before this can be properly connected to it #}
+                (Type)
             </a>
         </h3>
 
         <p class="search-results__list-card-description">
             {{ record.listing_description }}
         </p>
 
-        <dl class="tna-dl tna-dl--plain tna-dl--icon-padding tna-!--margin-bottom-s">
-            {% if record.held_by %}
-            <dt>
-                <i class="fa-solid fa-landmark"></i>
-                Held by
-            </dt>
-            <dd>{{ record.held_by }}</dd>
-            {% endif %}
-
+        <dl class="tna-dl tna-dl--plain tna-!--margin-bottom-s">
 
+            {% comment %}
+                Commented out as discussed on https://github.com/nationalarchives/ds-ohos-wagtail/pull/26
+                This will require some additional logic once we are working with OHOS data
+                {% if record.held_by %}
+                    <dt>Held by</dt>
+                    <dd>{{ record.held_by }}</dd>
+                {% endif %}
+            {% endcomment %}
 
             {% if record.date_created %}
-            <dt>
-                <i class="fa-solid fa-calendar"></i>
```

Comment:
same, please reinstate this as commented inside new dl, for the purpose of tna,non-tna buckets